rotkehlchen/tests/conftest.py

```diff
@@ -138,3 +140,44 @@ def requires_env(allowed_envs: list[TestEnvironment]):
         'CI' in os.environ and env not in allowed_envs,
         reason=f'Not suitable envrionment {env} for current test',
     )
+
+
+def get_cassette_dir(request: pytest.FixtureRequest) -> str:
+    return request.node.module.__name__
+
+
+@pytest.fixture
+def vcr_config(request: pytest.FixtureRequest, default_cassette_name: str) -> dict[str, Any]:
+    """
+    vcrpy config
+    - record mode: allow rewriting multiple cassettes using CASSETTE_REWRITE_PATH env variable
+      e.g. CASSETTE_REWRITE_PATH=rotkehlchen.tests.unit.decoders will rewrite all decoder cassettes
+    - decode compressed response: decode if not in CI (for readability)
+    """
+    rewrite_path = os.environ.get('CASSETTE_REWRITE_PATH')
+    cassette_dir = get_cassette_dir(request)
+    cassette_path = os.path.join(cassette_dir, default_cassette_name)
+
+    record_mode = "rewrite" if rewrite_path and cassette_path.startswith(rewrite_path) else "once"
+    decode_compressed_response = 'CI' not in os.environ
```

Comment:
fwiw it looks like GitHub Actions stores the cache directory as a [compressed](https://github.com/actions/toolkit/blob/main/packages/cache/src/cache.ts#L191) tar archive

Reply:
So no need to do extra compression, it would be faster to leave it to actions

Reply:
It's compressed by the HTTP server, this is an extra step to decompress